client/src/pages/Discovery/Discovery.tsx

```diff
@@ -31,11 +43,75 @@ export default function Discovery(): JSX.Element {
     });
   }, [updateSnackBarMessage]);
 
+  const handleSimpleSubmit = (
+    {
+      search,
+    }: {
+      search: string;
+    },
+    {
+      setSubmitting,
+    }: FormikHelpers<{
+      search: string;
+    }>,
+  ) => {
+    getAllContestsBySimpleSearch(search).then((data) => {
+      if (data.contest.length) {
```

Comment:
Send the response as `success` | `error` from the BE would make more sense :)

Reply:
modified contest controller